api/src/tests.rs

```diff
@@ -105,45 +105,7 @@ impl AccessModelStore for MockReconModelTest {
         Ok(None)
     }
 }
-#[tokio::test]
-async fn create_event() {
```

Comment:
let me know if you think it's worth it to try to revive this test.  Probably what I would do is take the hard-coded constants used in the unit tests in `api/src/server/event.rs` and pull them out into their own file and use them in both the unit tests in that file as well as in this unit test here.  It just feels like since the underlying store is mocked out anyway, this test isn't actually testing a whole lot of real functionality to begin with.  The actual coverage will come from the integration tests.

Reply:
oh wait, looking more deeply there is some fancy logic in the server code around passing the event into a channel to be processed by another Task.  That's probably worth having unit test coverage of.  Okay I'll go ahead with pulling those constants out and bringing this test back to life